### ISSUE TITLE
macOS: `alf_explicit_auths` table missing the sysprefs-listed application layer firewall exceptions 

### ISSUE THREAD

**Issue by arubdesu** [osquery/osquery#2322]
steps to repro: `osqueryi --line "select * from alf_explicit_auths"`
expected: a combination of the output from `/usr/libexec/ApplicationFirewall/socketfilterfw --listapps`, an example of which is 

```
ALF: total number of apps = 3 
1 :  /Users/abanks/Library/Application Support/Dropbox/Dropbox.app 
     ( Allow incoming connections ) 
2 :  /System/Library/CoreServices/RemoteManagement/ARDAgent.app 
     ( Allow incoming connections ) 
3 :  /Library/Application Support/Citrix/NetScaler Gateway.app 
     ( Allow incoming connections )
```

and also something like the cleaned-up output from `/usr/libexec/ApplicationFirewall/socketfilterfw -l`, which is loosely categorized in the sysprefs/security GUI as 'remote login' and 'remote management' listings. Instead, osquery lists:

```
process = org.python.python.app
process = com.apple.ruby
process = com.apple.a2p
process = com.apple.javajdk16.cmd
process = com.apple.php
process = com.apple.nc
process = com.apple.ksh
```

**Comment by Smjert** [comment]
Tested on latest master (d71420be93a35ad810646110e0e13c679a6521c3) on a macOS 10.14 and the issue is still present.

**Comment by GuillaumeRoss** [comment]
I can confirm I am seeing the same issue on Ventura public beta with 5.5.1.

**Comment by lucasmrod** [comment]
Table `alf_exceptions` contains the output of `/usr/libexec/ApplicationFirewall/socketfilterfw --listapps`.

@Smjert @zwass Shall we close this?

**Comment by directionless** [comment]
Fixed by #8434
